tests/sar.sh

```diff
@@ -3,5 +3,5 @@
 set -ex
 # The idiom "&& [ $? -gt 2 ] && true" ignores warnings from the previous command
 
-python --version 
+python --version
 python3.10 --version
```

Comment:
specifically python? Making sure it's linked to python3?

Reply:
These tests are effectively meaningless, and will almost certainly be going away during the Actions -> GHCR migration